include/openenclave/corelibc/endian.h

```diff
@@ -5,8 +5,8 @@
  * into oecore. It scopes down the determination of __BYTE_ORDER
  * relevant to memcpy avoiding GNUC/BSD specializations.
  */
-#ifndef _OE_ENDIAN_H
-#define _OE_ENDIAN_H
+#ifndef _OE_SYSCALL_ENDIAN_H
```

Comment:
This header remains in corelibc and should not require an updated header guard name.

Reply:
Thanks. Resolved.